holland/core/cmdshell.py

```diff
@@ -21,6 +21,11 @@
 
 LOGGER = logging.getLogger(__name__)
 
+holland_conf = '/etc/holland/holland.conf'
+currplatform = sys.platform
+if currplatform.startswith('freebsd'):
+    holland_conf = '/usr/local' + holland_conf
```

Comment:
I think using `sys.platform` directly is probably more idiomatic.  We're already using unixy paths, so maybe just use the simplest solution:

```
if sys.platform.startswith('freebsd'):
    holland_conf = '/usr/local/etc/holland/holland.conf'
else:
   holland_conf = '/etc/holland/holland.conf'
```